packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-swift-visitor.test.ts

```diff
@@ -2279,6 +2280,72 @@ describe('AppSyncSwiftVisitor', () => {
     });
   });
 
+  describe('Use Improved Pluralization', () => {
+    let wishSchema: string;
+    beforeEach(() => {
+      wishSchema = /* GraphQL */ `
```

Comment:
Nit: We can use the schema here to display the difference before and after the plural name change. Refer https://github.com/aws-amplify/amplify-cli/issues/8350
```
type NoteData
@model
{
    id: ID!
    name: String!
    description: String
    image: String
}

type Wishes @model 
@key(name: "byUsers", fields: ["idUser"]) 
{

  id: ID!
  date: AWSDateTime!
  idUser: ID!
  ev: ID! 
  deprecated: Boolean!  
  owner: String         
}

type Wish @model 
@key(name: "byUsers", fields: ["idUser"]) {

  id: ID!
  date: AWSDateTime!
  idUser: ID!
  ev: ID! 
  deprecated: Boolean!  
  owner: String         
}
```